docs/guides/circuit_runners.rst

```diff
@@ -0,0 +1,200 @@
+
+For more examples of using the :class:`MeasurementTrackingBackend <orquestra.quantum.runners.trackers.MeasurementTrackingBackend>`, refer to the `tracking backend tests <https://github.com/zapatacomputing/orquestra-quantum/blob/main/tests/orquestra/quantum/trackers_test.py>`_
+
+.. _integrated_circuit_runners:
+
+Currently integrated backends
+=============================
+
+Hardware backends
+-----------------
+
+* `QiskitRunner <https://github.com/zapatacomputing/orquestra-qiskit/blob/main/tests/orquestra/integrations/qiskit/runner/_qiskit_runner_test.py>`_ can wrap any Qiskit backend, including ones using physical hardware
+   * `A helper function for creating runners using IBMQ devices <https://github.com/zapatacomputing/orquestra-qiskit/blob/main/tests/orquestra/integrations/qiskit/runner/_ibmq_runner_test.py>`_
+* `BraketRunner <https://github.com/zapatacomputing/orquestra-braket/blob/main/tests/orquestra/braket/runner_test.py>`_
+
+Simulators
+----------
+
+* `SymbolicSimulator <https://github.com/zapatacomputing/orquestra-quantum/blob/main/tests/orquestra/quantum/symbolic_simulator_test.py>`_
+* `CirqSimulator <https://github.com/zapatacomputing/orquestra-cirq/blob/main/tests/orquestra/integrations/cirq/simulator/simulator_test.py>`_
+* `QiskitWavefunctionSimulator <https://github.com/zapatacomputing/orquestra-qiskit/blob/main/tests/orquestra/integrations/qiskit/runner/_qiskit_wavefunction_simulator_test.py>`_ can wrap any Qiskit backend supporting ``save_statevector`` instruction.
+
+* `QulacsSimulator <https://github.com/zapatacomputing/orquestra-qulacs/blob/main/tests/orquestra/integrations/qulacs/simulator_test.py>`_
+* `QSimSimulator <https://github.com/zapatacomputing/orquestra-cirq/blob/main/tests/orquestra/integrations/cirq/simulator/qsimsimulator_test.py>`_
+* `CuStateVecSimulator <https://github.com/zapatacomputing/orquestra-cirq/tree/main/src/orquestra/integrations/custatevec/simulator>`_
+* `BraketLocalSimulator <https://github.com/zapatacomputing/orquestra-braket/blob/main/tests/orquestra/braket/simulator/localbraket_test.py>`_
+* `braket_local_simulator <https://github.com/zapatacomputing/orquestra-braket/blob/main/tests/orquestra/braket/simulator/ondemandbraket_test.py>`_ function can be used for constructing local wavefunction simulator utilizing Braket's local simulator.
+
+Conversions to other frameworks
+===============================
+
+The integrations Orquestra Core has with :ref:`many other frameworks <orq_core_structure>` not only allows for running on Orquestra-supported backends, but also allows for converting circuits to and from those frameworks. This process is very fast so converting to and from different frameworks can be done multiple times with negligible overhead.
```

Comment:
`This process is very fast so converting to and from different frameworks can be done multiple times with negligible overhead.` 
Well... not *always* true, so perhaps I would add "for most purposes"?